pkg/database/db_test.go

```diff
@@ -24,7 +25,11 @@ func TestDatabaseParseURL(t *testing.T) {
 	a := assert.New(t)
 	driver, dataSource, e := ParseURL(testingMySQLURL())
 	a.EqualValues(driver, "mysql")
-	a.EqualValues(dataSource, "root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0")
+	user := getEnv("SQLFLOW_TEST_DB_MYSQL_USER", "root")
```

Comment:
Can we reuse 

https://github.com/sql-machine-learning/sqlflow/blob/44939623ee243d06f5d7b9f3dac4e8c5d8b7c4b3/pkg/database/testing.go#L76-L84

Reply:
Sure, it's a good point.